### DISCUSSION TITLE
No Proper Rendering on Custom User Agent?

### DISCUSSION THREAD

**Discussion by yilmaz-sp** [preactjs/preact#3021] [General]
Hi. We tried to render a website on a custom mobile browser, which maintains a custom user agent. It resulted in some of the content, mainly images, not being rendered. Is this an expected thing? What are the trusted browsers (eg Safari, Chrome...)?

**Comment by JoviDeCroock** [comment]
I don't think this has anything to do with Preact, this will be some setting for your image-cdn that is tailored to UA's i'd imagine.

**Reply by marvinhagemeister** [reply]
A good tip is to open the Browsers developer tools and check the network tab. If the image requests are there, but failing than it is a 99.9% chance that the problem is with the server/CDN

**Reply by yilmaz-sp** [reply]
Yes, I better check this out 👍
